cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -213,7 +213,7 @@
 
     {# TODO: Remove Else conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% if page and page.media %}
+    {% if page %}
```

Comment:
cool, I would just test this on a Browse page because that is a Wagtail page with no `page.media` object.